src/roqueria/main.py

```diff
@@ -0,0 +1,79 @@
+import csv
+from dataclasses import dataclass
+from enum import Enum
+
+
+class Plan(str, Enum):
+    CASUAL = "CASUAL"
+    REGULAR = "REGULAR"
+    PRO = "PRO"
+    DELETED = "DELETED"
+
+
+@dataclass(frozen=True)
+class User:
+    name: str
+    birthdate: str  # should be a datetime.date
+    plan: Plan
+
+
+# Does it make sense to create this as a helper function?
+# Should it be a private function?
+def _parse_csv_user(unparsed_user: list) -> User:
+    user = User(name=unparsed_user[0], birthdate=unparsed_user[1], plan=unparsed_user[2])
+
+    return user
+
+
+def create_user(name: str, birthdate: str, plan: Plan) -> int:
+    path = "data/users"
+    new_user = User(name, birthdate, plan)
+
+    with open(path, "r+") as file:
+        csv_reader = csv.reader(file)
+        user_id = sum(1 for _ in csv_reader)
+
+        csv_writer = csv.writer(file)
+        csv_writer.writerow(
+            [new_user.name, new_user.birthdate, new_user.plan]  # not a big improvement, but checks the type
+        )
+
+    return user_id
+
+
+def get_user(id: int) -> User:
+    user_file = "data/users"
+    with open(user_file, "r") as file:
+        csv_reader = csv.reader(file)
+        for _ in range(id):
```

Comment:
What happens if a pass an invalid or unused ID? What happens if I pass -1? There is only a single test for the happy path

Reply:
Is -1 a wrong value? Should we allow it if someone wants to find the last record?